domain/consensus/processes/coinbasemanager/block_reward_switch_test.go

```diff
@@ -88,34 +88,34 @@ func TestBlockRewardSwitch(t *testing.T) {
 		}
 
 		// Add another block. We expect it to be another pruning point
-		lastPruningPointHash, _, err := tc.AddBlock([]*externalapi.DomainHash{tipHash}, nil, nil)
-		if err != nil {
-			t.Fatalf("AddBlock: %+v", err)
-		}
+		//lastPruningPointHash, _, err := tc.AddBlock([]*externalapi.DomainHash{tipHash}, nil, nil)
+		//if err != nil {
```

Comment:
Can you delete this test?

Reply:
sure, kept it because we might change the logic later and will want to reuse those tests

Reply:
We removed the related logic and I don't see why we'll revert to random monetary policy any time soon.
Anyway, it still exists in git

Reply:
Done